.github/workflows/chef-test.yml

```diff
@@ -37,13 +37,35 @@ jobs:
         run: |
           make rake-spec
 
+  chef-kitchen-matrix:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Get linux distros and suites
+        id: get-linux-matrix
+        run: |
+          distros=$(yq '[.platforms[] | "\"" + (.name | sub("_", "-") | sub("\.", "")) + "\""] | join(", ")' kitchen.yml)
```

Comment:
Does it make sense to add a helper script to generate these vars that can be sourced instead of requiring an action run? It could contain the same invocations or maybe generates a static file that changes on platform/suite additions.

Reply:
on second thought since the matrix is the only use case it probably doesn't make sense unless the content would be consumed by an additional process, but there's the small risk of uncaught yq issues by not having the intermediary format.

Reply:
Yeah, we just want to create the `matrix` json objects for the downstream matrix configs at run time.  I'll add some checks to make sure that `distros` and `suites` are not empty.